### PR TITLE
Task 3 final approval needed.

### PR DESCRIPTION
**Pull request by vminin-solvd**
I will add more business logic to my project as I do task 4 on a new branch. In the meantime I have made the changes you pointed out.

### CODE REVIEW

File: zoo/animal/IAttack.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package zoo.animal;
+
+public interface IAttack {
+    void attack();
+}
```
Comment (ankochergin) [inline zoo/animal/IAttack.java]:
```suggestion
public interface IAttack {

    void attack();
    
}
```

File: zoo/Main.java [hunk 2]
```diff
@@ -48,6 +52,9 @@ public static void main(String[] args) {
         tiger.setLocation("Tiger's Den");
         zooKeeper.setLocation("Tiger's Den");
 
+        tiger.showSpecies();
+
+
```
Comment (ankochergin) [inline zoo/Main.java]:
```suggestion
        tiger.showSpecies();

```

Comment (ankochergin) [reply zoo/Main.java]: no more than 2 spaces

**Comment by vminin-solvd** [comment]
I have committed more changes.

**Pull request merged**